packages/aws-cdk-lib/aws-events-targets/lib/log-group.ts

```diff
@@ -78,6 +79,14 @@ export interface LogGroupProps extends TargetBaseProps {
    * @default - the entire EventBridge event
    */
   readonly logEvent?: LogGroupTargetInput;
+
+  /**
+   * Automatically configure an AWS CloudWatch Log Group Resource Policy for Events Target.
+   *
+   * @default true
+   * @default - false if `@aws-cdk/aws-events:eventsTargetDisableLogGroupResourcePolicy` is enabled, true otherwise
+   */
+  readonly logGroupResourcePolicy?: boolean;
```

Comment:
```suggestion
  /**
   * Automatically configure an AWS CloudWatch Log Group Resource Policy for Events Target.
   *
   * Note: if set to true, you must create and configure a CloudWatch Log Group Resource Policy separately.
   *
   * Can be used to avoid hitting the limit of 10 CloudWatch Log Group Resources in an AWS environment.
   *
   * @default false
   */
  readonly disableLogGroupResourcePolicyCreation?: boolean;
```
Open to discussion on the prop name + doc string, but I think this is closer to what we want to convey.